sw/device/lib/runtime/hart.h

```diff
@@ -23,7 +24,11 @@
  *
  * This function may behave as if it is a no-op.
  */
-inline void wait_for_interrupt(void) { asm volatile("wfi"); }
+inline void wait_for_interrupt(void) {
+#ifdef OPENTITAN_PLATFORM_RV32
```

Comment:
I don't see this macro in our codebase

Reply:
It should be OT_PLATFORM_RV32. :facepalm: